### PR TITLE
feat: allow the swagger path to be configurable

### PR DESCRIPTION
**Pull request by Junjiequan**
## Description

This PR makes the Swagger path configurable by allowing it to be set through the SWAGGER_PATH environment variable. If not provided, it defaults to “explorer.”

## Motivation

Making the Swagger path configurable improves flexibility, allowing different paths for different environments 

## Fixes:
Please provide a list of the fixes implemented by this PR

* Items added

## Changes:
Please provide a list of the changes implemented by this PR

* changes made

## Tests included

- [ ] Included for each change/fix?
- [ ] Passing? (Merge will not be approved unless this is checked) 

## Documentation
- [ ] swagger documentation updated \[required\]
- [ ] official documentation updated \[nice-to-have]

### official documentation info
If you have updated the official documentation, please provide PR # and URL of the pages where the updates are included

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -36,8 +36,9 @@ async function bootstrap() {
       docExpansion: "none",
     },
   };
+  const swaggerPath = process.env.SWAGGER_PATH || "explorer";
```
Comment (nitrosx) [inline src/main.ts]: Can you place this in the config service, please?

**Review by nitrosx — CHANGES_REQUESTED** [review]
Good to go, just one minor request

File: src/config/configuration.ts [hunk 2]
```diff
@@ -58,6 +58,10 @@ const configuration = () => {
   });
 
   const config = {
+    VERSIONS: {
+      API: "v3",
+    },
```
Comment (nitrosx) [inline src/config/configuration.ts]: Do you mind using camel case, so we are consistent with all the other entries?

File: src/main.ts [hunk 3]
```diff
@@ -15,6 +15,11 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule, {
     bufferLogs: true,
   });
+  const configService: ConfigService<Record<string, unknown>, false> = app.get(
+    ConfigService,
+  );
+  const apiVersion = configService.get<string>("VERSIONS.API");
```
Comment (nitrosx) [inline src/main.ts]: Camel case?

**Review by nitrosx — CHANGES_REQUESTED** [review]
To minor changes

File: src/main.ts (hunk 1 above)
Comment (Junjiequan) [reply src/main.ts]: fixed

File: src/main.ts (hunk 3 above)
Comment (Junjiequan) [reply src/main.ts]: fixed

File: src/config/configuration.ts (hunk 2 above)
Comment (Junjiequan) [reply src/config/configuration.ts]: fixed

**Review by nitrosx — APPROVED** [review]

**Pull request merged**